<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_Check</class>
 <widget class="QDialog" name="Data_Check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1241</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">QTableView {
    border: 1px solid #ccc;
    border-radius: 5px;
}

QTableView::item {
    padding: 5px;
}

QTableView::item:selected {
    background-color: #f0f0f0;
}

QHeaderView::section {
    background-color: #f0f0f0;
    color: #333;
    font-weight: bold;
    padding: 5px;
}

QTableView::item:hover {
    background-color: #e0e0e0;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="currentText">
           <string>车牌号</string>
          </property>
          <item>
           <property name="text">
            <string>车牌号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ID卡号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>未出的车</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>已出的车</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>花费金额</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>650</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>出入车辆图片</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>入库</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>出库</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
